QUEST_LV_0100.tsv

```diff
@@ -5082,39 +5082,39 @@ QUEST_LV_0100_20150317_005077	Follower Donatas told you to check the central bar
 QUEST_LV_0100_20150317_005078	You defeated Malletwyvern at the central altar. Talk to Follower Donatas.
 QUEST_LV_0100_20150317_005079	Defeat Malletwyvern
 QUEST_LV_0100_20150317_005080	Talk to Follower Tomas
-QUEST_LV_0100_20150317_005081	Follower Tomas is seeking for somebody's help on the 1st floor of Tenet Chapel.
-QUEST_LV_0100_20150317_005082	Defeat Yognome
-QUEST_LV_0100_20150317_005083	Follower Tomas told you to defeat Yognomes that are walking around here since he can't concentrate his work.
-QUEST_LV_0100_20150317_005084	As Follower Tomas requested to you, you defeated all Yognomes. Return to Follower Tomas.
+QUEST_LV_0100_20150317_005081	Follower Tomas wants help on the 1st floor of Tenet Chapel.
+QUEST_LV_0100_20150317_005082	Defeat Yognomes
+QUEST_LV_0100_20150317_005083	Follower Tomas told you to defeat the Yognomes around here, since they are trying to eat the altar and he can't concentrate his work.
+QUEST_LV_0100_20150317_005084	As Follower Tomas requested of you, you defeated all the Yognomes. Return to Follower Tomas.
 QUEST_LV_0100_20150317_005085	Collect the Eyes of Madness
-QUEST_LV_0100_20150317_005086	Follower Tomas told you to defeat Rodelin and bring the Eyes of Madness to protect the Paladin followers and the altar from the demons.
-QUEST_LV_0100_20150317_005087	Follower Tomas collected enough Eyes of Madness. Return to Follower Tomas.
-QUEST_LV_0100_20150317_005088	Defeat Rodelin and obtain %s
+QUEST_LV_0100_20150317_005086	Follower Tomas told you to defeat Rodelins and retrieve their Eyes of Madness, which can be used to hide from demons.
+QUEST_LV_0100_20150317_005087	You collected enough Eyes of Madness. Return to Follower Tomas.
+QUEST_LV_0100_20150317_005088	Defeat Rodelins and obtain %s
 QUEST_LV_0100_20150317_005089	Obtain the Eyes of Madness
-QUEST_LV_0100_20150317_005090	Follower Tomas is seeking for somebody's help on the 1st floor of Tenet Chapel.
-QUEST_LV_0100_20150317_005091	Follower Tomas told you to defeat Unknocker beside the barrier first. Attack Unknocker when Follower Tomas lures it.
-QUEST_LV_0100_20150317_005092	You defeated the Unknocker that threatened the barrier. Talk to Follower Tomas again.
+QUEST_LV_0100_20150317_005090	Follower Tomas wants help on the 1st floor of Tenet Chapel.
+QUEST_LV_0100_20150317_005091	Follower Tomas told you to defeat the Unknocker that is guarding the altar needed to disable the barrier. Attack Unknocker when Follower Tomas lures it.
```

Comment:
Follower Tomas told you that it is necessary to defeat the Unknocker that is guarding the altar in order to disable the barrier.